src/de/teamgrit/grit/preprocess/MailPreprocessor.java

```diff
@@ -71,19 +71,21 @@ public static PreprocessingResult preprocess(
         String loginUsername = connection.getUsername();
         String loginPassword = connection.getPassword();
         String mailServer = connection.getLocation();
+        String protocol = connection.getProtocol();
+        String allowedDomain = connection.getAllowedDomain();
 
         List<Submission> submissions = null;
 
         List<String> structureList = new LinkedList<>();
         structureList.add("TOPLEVEL");
         structureList.add(".*@.*");
-        structureList.add(exerciseName.toString());
+        structureList.add(exerciseName.toString().replaceAll("\\s", "_"));
```

Comment:
String Bereinigung in eigene Methode auslagern, dann lassen sich einfach Fälle hinzufügen.